src/praisonai/praisonai/bots/_delivery.py

```
@@ -0,0 +1,564 @@
+            typing: Whether to show typing indicator
+            **kwargs: Additional platform-specific arguments
+            
+        Returns:
+            List of sent messages (may be multiple due to chunking)
+        """
+        caps = self.capabilities
+        text = content if isinstance(content, str) else str(content)
+        
+        # Show typing indicator if supported
+        if typing and caps.supports_typing:
+            asyncio.create_task(self._show_typing(channel_id))
```

Ruff (0.15.17)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)